bot/exts/evergreen/avatar_modification/avatar_modify.py

```diff
@@ -121,6 +121,50 @@ async def eightbit_command(self, ctx: commands.Context) -> None:
 
         await ctx.send(embed=embed, file=file)
 
+    @avatar_modify.group(name="reverse", root_aliases=['reverse'])
+    async def reverse(self, ctx: commands.Context) -> None:
+        """Group for the reverse commands."""
+        if not ctx.invoked_subcommand:
+            await invoke_help_command(ctx)
+
+    @reverse.command(name="text")
+    async def reverse_text(self, ctx: commands.Context, *, text: str) -> None:
+        """Sends the given text backwards."""
+        await ctx.send(f"> {text[::-1]}", allowed_mentions=discord.AllowedMentions.none())
+
+    @reverse.command(name="image")
+    async def reverse_photo(self, ctx: commands.Context) -> None:
```

Comment:
I feel like this should be the default behaviour of `.reverse` so if a user doesn't so `.reverse text <some text>` it defaults to reversing their avatar, rather than requiring them to do `.reverse image`

Reply:
Merged reverse into one command, resolved